issue-rules.yml

```diff
@@ -339,7 +339,7 @@ nomatches:
 
 # always runs after rules.  look for missing or invalid sets of tags
 tags:
-- noneIn: ['bug', 'enhancement', 'question']
+- noneIn: ['bug', 'enhancement', 'question', 'DTS']
   addLabels: ['triage']
 - noneMatch: '\s*Area:\s*([^]*)'
   addLabels: ['route']
```

Comment:
To avoid getting the `route` tag, we could change this from `noneMatch` to a `noneIn: ['Area: ABTT', 'Area: Artifacts', (...)]`. That doesn't need to happen in this pr, but we can try to follow up.